nettests/groups.go

```diff
@@ -39,6 +39,7 @@ var NettestGroups = map[string]NettestGroup{
 	"circumvention": {
 		Label: "Circumvention Tools",
 		Nettests: []Nettest{
+			STUNReachability{},
```

Comment:
Doesn't this mean that we are going to be in fact recording (and displaying) the result for the STUNReachability test in the UI as well?

I thought the plan was to start collecting data for this test, but without presenting it in the UI.

Reply:
This runs the test as part of the nettest group. We will show the results when running from the command line, but I don't know whether the desktop app will show this results and in which way. I would assume it will skip results it doesn't know?

Reply:
I believe the desktop app is going to either display an error in the row where this is shown or potentially even crash.

@sarathms perhaps know better the behavior.

I think the easiest way to go about implementing this is to hide completely the output of the test also from probe-cli, that way we don't have to worry about the integration with the desktop app as the output is entirely identical.

Reply:
I would like to avoid writing hackish code for running this experiment. Ideally the desktop app should only process the measurement it understands. Imma try and run circumvention with a OONI Probe that supports stun_reachability and see what happens.

Reply:
Resolving this conversation: we're not running the experiment until we have a UI.